src/metabase/api/table.clj

```diff
@@ -57,6 +57,7 @@
   "Get all `Tables` visible to the current user which were created by uploading a file."
   []
   (as-> (t2/select Table, :active true, :is_upload true, {:order-by [[:name :asc]]}) tables
+        (map #(update % :schema str) tables)
```

Comment:
This might merit a comment / link to Kamil's bug report

Reply:
Pulling this into a newer branch I'm working on to avoid another roundabout with CI.